ui/src/app/workflows/components/workflows-row/workflows-row.tsx

```diff
@@ -94,7 +94,8 @@ export class WorkflowsRow extends React.Component<WorkflowsRowProps, WorkflowRow
                         </div>
                         <div className='columns small-1'>{isArchivedWorkflow(wf) ? 'true' : 'false'}</div>
                         {(this.props.columns || []).map(column => {
-                            const value = wf.metadata?.labels[column.key];
+                            // best not to make any assumptions and wait until this data is filled
+                            const value = wf?.metadata?.labels ? wf.metadata.labels[column.key] : 'unknown';
```

Comment:
```suggestion
                            const value = wf?.metadata?.labels?.[column.key] ?? 'unknown';
```

I believe this is the more conventional syntax nowadays with some slightly nicer properties

Though I think with `?.[column.key]`, `'unknown'` would technically no longer be required, so could also also do:

```suggestion
                            const value = wf?.metadata?.labels?.[column.key];
```